editors/atlas-scope-editor/editor.tsx

```diff
@@ -113,7 +112,7 @@ export default function Editor(props: IProps) {
                 className="w-full"
                 name="docNo"
                 label="Doc No"
-                onChange={(e) => {
+                onBlur={(e) => {
                   props.dispatch(actions.setDocNumber({ docNo: e.target.value }));
```

Comment:
dispatching actions directly in the `onBlur`/`onChange` could lead to dirty history in the models, you should dispatch changes if something actually changed. Instead of doing this, use the Form features `submitChangesOnly` and submit the form manually in those methods

Reply:
you are right done